k8s/namespaces/ia/ia-case-api/aat.yaml

```diff
@@ -2,10 +2,8 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: ia-case-api
-spec:
   values:
     java:
-      image: hmctspublic.azurecr.io/ia/case-api:pr-1152
       environment:
         IA_TIMED_EVENT_SERVICE_ENABLED: "true"
         IA_HOME_OFFICE_INTEGRATION_ENABLED: "true"
```

Comment:
Also need to remove the below env props added for this testing:
paymentAfterRemissionRejection.dueInMinutes: 1440
paymentEaHuNoRemission.dueInMinutes: 1440